### PR TITLE
update to new Backend API Close #14

### LINKED ISSUE #14: Implement backend REST API and Database
CRUD operations for user, word

### PR DESCRIPTION
**Pull request by yugan2005**

### CODE REVIEW

File: site.webmanifest [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (ganneo) [inline site.webmanifest:1]: Add another line at the bottom.

File: src/service/UserService.js [hunk 2]
```diff
@@ -1,56 +1,53 @@
...
   }
 
-  async addUser(user) {
-    const plainObj = await this.userClient.add(UserService.hashPassword(user));
-    if (!plainObj) {
-      return null;
+  async saveUser(user) {
+    if (!user.id) {
+      user.password = md5(user.password);
     }
```
Comment (ganneo) [inline src/service/UserService.js]: How about when a user changes his password? It should be hashed.

**Review by ganneo — CHANGES_REQUESTED** [review]

**Review by ganneo — APPROVED** [review]
LGTM

**Review by tiffanygan — APPROVED** [review]
LGTM

**Pull request merged**